zuul.d/projects.yaml

```diff
@@ -18,7 +18,7 @@
     default-branch: main
 
 - project:
-    name: ^(github.com/|)(ansible-collections|sap-linuxlab)/community\..*
+    name: ^(github.com/|)(ansible-collections|sap-linuxlab|openshift)/community\..*
```

Comment:
Looks good to me. I would personally suggest to use a different line just for clarity.

Reply:
ah I was about to change it, but it has already been merged
Thanks